src/main/target/FORTINIF4/target.h

```diff
@@ -0,0 +1,161 @@
+#define LED0                    PB5
+#define LED1                    PB6
+#define LED_STRIP
+/*---------------------------------*/
+
+/*------------BEEPER---------------*/
+#define BEEPER                  PB4
+#define BEEPER_INVERTED
+/*---------------------------------*/
+
+/*------------SENSORS--------------*/
+// MPU interrupt
+#define USE_EXTI
+#define MPU_INT_EXTI            PC4
+#define EXTI_CALLBACK_HANDLER_COUNT 1 // MPU data ready
+//#define DEBUG_MPU_DATA_READY_INTERRUPT
+#define USE_MPU_DATA_READY_SIGNAL
+#define ENSURE_MPU_DATA_READY_IS_LOW
+
+#define MPU6000_CS_PIN          PA8
+#define MPU6000_SPI_INSTANCE    SPI1
+
+#define MPU6500_CS_PIN          PA8
+#define MPU6500_SPI_INSTANCE    SPI1
+
+#define ICM20689_CS_PIN         PA8
+#define ICM20689_SPI_INSTANCE   SPI1
+
+#define GYRO
+#define USE_GYRO_SPI_ICM20689
+#define GYRO_ICM20689_ALIGN     CW180_DEG
+
+#define USE_GYRO_SPI_MPU6000
+#define GYRO_MPU6000_ALIGN      CW180_DEG
+#define USE_ACC_SPI_MPU6000
+#define ACC_MPU6000_ALIGN       CW180_DEG
+
+#define USE_GYRO_MPU6500
+#define USE_GYRO_SPI_MPU6500
+#define GYRO_MPU6500_ALIGN      CW180_DEG
+
+#define ACC
+#define USE_ACC_SPI_ICM20689
+#define ACC_ICM20689_ALIGN      CW180_DEG
+
+#define USE_ACC_SPI_MPU6000
+#define ACC_MPU6000_ALIGN       CW180_DEG
+
+#define USE_ACC_MPU6500
+#define USE_ACC_SPI_MPU6500
+#define ACC_MPU6500_ALIGN       CW180_DEG
+
+/*---------------------------------*/
+
+/*------------FLASH----------------*/
+#define M25P16_CS_PIN           PB3
+#define M25P16_SPI_INSTANCE     SPI3
+
+#define USE_FLASHFS
+#define USE_FLASH_M25P16
+/*---------------------------------*/
+
+/*-----------USB-UARTs-------------*/
+#define USE_VCP
+//#define VBUS_SENSING_PIN PA8
+//#define VBUS_SENSING_ENABLED
+
+#define USE_UART1
+#define UART1_RX_PIN            PA10
+#define UART1_TX_PIN            PA9
+
+#define USE_UART3
+#define UART3_RX_PIN            PB11
+#define UART3_TX_PIN            PB10
+#define INVERTER_PIN_UART3      PC15
+
+#define USE_UART4
+#define UART4_TX_PIN            PA0
+#define UART4_RX_PIN            PA1
+
+#define USE_UART6
+#define UART6_RX_PIN            PC7
+#define UART6_TX_PIN            PC6
+
+#define SERIAL_PORT_COUNT       5
+//SPECKTRUM BIND
+#define SPEKTRUM_BIND_PIN       UART3_RX_PIN
+
+#define CMS
+#define USE_MSP_DISPLAYPORT
+/*---------------------------------*/
+
+/*-------------SPIs----------------*/
+#define USE_SPI
+
+#define USE_SPI_DEVICE_1
+#define SPI1_NSS_PIN            PA8
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+#define USE_SPI_DEVICE_3
+#define SPI3_NSS_PIN            PB3
+#define SPI3_SCK_PIN            PC10
+#define SPI3_MISO_PIN           PC11
+#define SPI3_MOSI_PIN           PC12
+/*---------------------------------*/
+
+/*-------------ADCs----------------*/
+#define USE_ADC
+#define VBAT_ADC_PIN            PC2
+#define CURRENT_METER_ADC_PIN   PC1
+#define BOARD_HAS_VOLTAGE_DIVIDER
```

Comment:
Not supported any more. Use `DEFAULT_VOLTAGE_METER_SOURCE` / `DEFAULT_CURRENT_METER_SOURCE` instead.